MLInference.py

```diff
@@ -56,10 +57,22 @@ def ACPO_LOG(msg):
 
 field_name_set = {
     "modelname", "features", "outputs", "signature", "modeldirectory",
-    "outputkey", "modelinference"
+    "outputkey", "modelinference", "loadmodeltype"
 }
 
-def get_imported_and_infer(model_dir, signature):
+
+def get_torch_imported_and_infer(model_dir, signature):
+  """
+  Return the imported model and using the
+  given model_dir
+  """
+  model_path = os.path.join(os.path.dirname(__file__), model_dir)
+  torch_model_name = "modelfi.pt"
```

Comment:
Why is this hardcoded? I am pretty sure that the name of the model can be stored in the ACPO file and accessed here appropriately. Lets make this properly data-driven.